portal/templates/scheduled_jobs_list.html

```diff
@@ -49,20 +54,82 @@ <h4>{{ _("Available Tasks") }}</h4>
 function triggerJob(event, jobId) {
```

Comment:
Would it be reasonable to move this to a separate JS file? Trying to move away from inline JS so we can prevent XSS

Reply:
I'm fine with that, but I'm not sure what our standard approach here is (aka where exactly this code should live instead). Maybe @achen2401 can take a look?

Reply:
@OptimusRhine adam, the JS code here probably merits its own JS file since it only relates to schedule job.  I, however, don't have strong opinion here as since only admin staff has access to schedule job-related pages - and I think a check on user role should probably prevent any other user from accessing those pages, but  I am not an expert on XSS attack. I will leave it up to you whether to make the change here or not.

Reply:
I think it's fine to leave as-is for now, as the page wouldn't be loaded in the first place unless the user has admin access.

Reply:
@OptimusRhine Sounds like a worthy goal to pull the JS out, if it's not too much to ask?

Reply:
Thanks! And OK, since it sounds like we're going for consistency with all our JS, I'll go ahead and pull that out.